src/current/v23.2/physical-cluster-replication-technical-overview.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Physical Cluster Replication Technical Overview
+summary: Understand the technical details of CockroachDB physical cluster replication.
+toc: true
+docs_area: manage
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} Physical cluster replication uses two clusters with _cluster virtualization_ enabled to stream data from an active primary cluster to a passive standby cluster. Each cluster contains: a _system interface_ and an application _virtual cluster_: {% comment %}Link to Matt's work here on general cluster virtualization reference{% endcomment %}
```

Comment:
```suggestion
{% include_cached new-in.html version="v23.2" %} Physical cluster automatically and continuously streams data from an active _primary_ CockroachDB cluster to a passive _standby_ cluster. Each cluster contains: a _system interface_ and an application _virtual cluster_: {% comment %}Link to Matt's work here on general cluster virtualization reference{% endcomment %}
```
WDYT? This loses a tiny bit of resolution but I think it is easier to parse. "two clusters with CV enabled" seems like it is covered in the other topics.

Reply:
I like it, I have adapted to this!